lib/mongo/bulk_write.rb

```diff
@@ -324,9 +324,15 @@ def can_hint?(connection)
     # and some which are not), in which case the driver expects the server to
     # fail the operation with an error.
     #
-    # @raise [ Error::InvalidUpdateDocument, Error::InvalidReplacementDocument ]
+    # Raise an ArgumentError if requests is empty.
+    #
+    # @raise [ Error::InvalidUpdateDocument, Error::InvalidReplacementDocument,
+    #   ArgumentError ]
     #   if the document is invalid.
     def validate_requests!
+      if @requests.empty?
+        raise ArgumentError, "Bulk write requests cannot be empty."
```

Comment:
Exception messages in Ruby do not (generally) have the period at the end.

Reply:
done